api/handlers/delete_user.go

```diff
@@ -13,24 +11,10 @@ type deleteUser struct{
 func NewDeleteUser(rt *ShivaniCustomServerExample1.Runtime) users.DeleteUserHandler{
 	return &deleteUser{rt:rt}
 }
-
-// func (d *deleteUser) Handle(del users.DeleteUserParams) middleware.Responder{
-// 	//if the val returing from DeleteUser = nil then return NewDestroyOneNoContent else not nil then return 404
-// 	isError:=d.rt.GetManager().DeleteUser(del.ID)
-// 	if isError == nil{
-// 		res:=users.NewDeleteUserNoContent()
-// 		return res
-// 	}
-// 	res:=users.NewDeleteUserNotFound().WithPayload(isError)
-// 	return res
-// }
-
 func (d *deleteUser) Handle(del users.DeleteUserParams) middleware.Responder{
 	err :=d.rt.GetManager().DeleteUser(del.ID)
```

Comment:
this can be simplify with inline 
if err := <calll>; err!nil{
// code block 
}